package.json

```diff
@@ -7,13 +7,15 @@
     "preversion": "yarn test",
     "benchmark": "node benchmark",
     "eslint": "eslint \"*.js\" benchmark test",
+    "tslint": "eslint test/typescript/index.tsx",
     "test-browser": "karma start test/browser/karma.conf.js",
     "test-browser-ie": "karma start test/browser/karma.conf.ie.js",
     "test-node": "mocha \"test/node/*.spec.js\"",
-    "test-node-cov": "nyc mocha \"test/node/*.spec.js\"",
-    "test-ts": "tsc --target ES5 --noImplicitAny index.d.ts",
-    "test": "builder concurrent --buffer eslint test-ts test-node-cov test-browser",
-    "test-ie": "builder concurrent --buffer eslint test-ts test-node-cov test-browser-ie",
+    "test-node-cov": "nyc yarn test-node",
```

Comment:
why does the `nyc` work show up on this PR?

Reply:
This needs to be:

```js
"test-node-cov": "nyc mocha \"test/node/*.spec.js\"",
```